code/game/turfs/open.dm

```diff
@@ -195,14 +195,14 @@
 /turf/open/proc/freon_gas_act()
 	for(var/obj/I in contents)
 		if(I.resistance_flags & FREEZE_PROOF)
-			return
+			break
```

Comment:
```suggestion
			continue
```

It's still a bit weird for a single object to block freezing of other objects on the tile

Reply:
Contents within objects such as sec cams have freeze proofing so it returns and stops the proc from continuing.